pkg/controller/certapi/certapi.go

```diff
@@ -76,29 +77,33 @@ func (c *Controller) validateToken(ctx context.Context, verToken string, publicK
 		kidHeader := token.Header[verifyapi.KeyIDHeader]
 		kid, ok := kidHeader.(string)
 		if !ok {
-			return nil, fmt.Errorf("missing 'kid' header in token")
+			message := "missing 'kid' header in token"
```

Comment:
Kinda nitty, but I think these would all be better as:

```go
err := fmt.Errorf("something something something")
logger.Infow("thing", "error", err)
return nil, err
```

You can pass an error to the logger